contracts/mocks/EthereumProverTest.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.7.3;
+
+import "../EthereumProver.sol";
+
+contract EthereumProverTest {
```

Comment:
Is there a corresponding test in javascript somewhere?

Reply:
submitted by mistake, will remove it.